### PR TITLE
Promote incubator/logstash to stable

### PR DESCRIPTION
**Pull request by rendhalver**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/logstash/README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [Logstash](https://www.elastic.co/products/logstash) is an open source, server-side data processing pipeline that ingests data from a multitude of sources simultaneously, transforms it, and then sends it to your favorite “stash.”
 
+**Note - this chart has been deprecated and [moved to stable](../../stable/burrow)**.
```
Comment (davidkarlsen) [inline incubator/logstash/README.md]: why burrow?

Comment (christian-roggia) [reply incubator/logstash/README.md]: I assume this is a Ctrl+C Ctrl+V gone wrong

Comment (rendhalver) [reply incubator/logstash/README.md]: oops.
Stoopid cut and paste

**Comment by rendhalver** [comment]
@christian-roggia It seems that we need ltgm and approve tags before this can be merged.
@unguiculus Is that intentional or am I reading that incorrectly?

**Comment by cpanato** [comment]
@rendhalver the owners should /approve /lgtm since this chart there is no owners file yet, this need to be approved by the owners from the chart repo

hope it clarifies your questions

**Comment by rendhalver** [comment]
Ah that explains it @cpanato thank you.

File: stable/logstash/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+description: Logstash is an open source, server-side data processing pipeline
+icon: https://www.elastic.co/assets/blt86e4472872eed314/logo-elastic-logstash-lt.svg
+home: https://www.elastic.co/products/logstash
+name: logstash
+version: 0.8.3
```
Comment (davidkarlsen) [inline stable/logstash/Chart.yaml:6]: set version to 1.0.0 in the promoted repo

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by davidkarlsen** [comment]
@rendhalver can you fix the merge conflict (and get that chart-change over to the stable tree?)

**Comment by rendhalver** [comment]
Yep. I will get that sorted when I can.

**Comment by rendhalver** [comment]
Closing in favour of #8479

**Pull request closed without merging**